src/main/kotlin/no/nav/medlemskap/regler/common/Resultat.kt

```diff
@@ -10,6 +10,6 @@ data class Resultat(
         val begrunnelse: String = "",
         val svar: Svar,
         var harDekning: Svar? = null,
-        var dekning: List<String> = listOf(),
+        var dekning: String = "",
```

Comment:
Hvorfor er harDekning og dekning var, ikke val? Skal de kunne endres i Resultat etter at Resultat er opprettet?

Reply:
Ja, jeg endrer Resultat i Regel-klassen. Hvis jeg har forstått det rett måtte jeg lagd egne konklusjons-metoder for dekning, men likte denne måten bedre. Mulig jeg har oversett en bedre løsning som tillater meg å bruke val?